Gemfile

```diff
@@ -8,10 +8,15 @@ group :development, :test do
   gem 'pry' unless ENV['CI']
   gem 'pry-byebug' unless ENV['CI']
   gem "simplecov", require: false
+  # Override the coffee-rails version in engine_cart for rails 4.2
+  case ENV['RAILS_VERSION']
+  when /^4.2/
+    gem 'coffee-rails', '~> 4.2.0'
```

Comment:
> The line-44 code is maintained by engine-cart, it should not be modified by anything but engine cart

I wouldn't go that far -- instead, I'd say: by modifying it, the onus is on the project maintainers to ensure it is maintained the way they expect.

I thought having duplicate gem declarations (particularly conflicting ones) makes bundler unhappy?